src/main/java/seedu/address/model/room/RoomIsOccupiedPredicate.java

```diff
@@ -0,0 +1,19 @@
+package seedu.address.model.room;
+
+import java.util.function.Predicate;
+
+/**
+ * Tests that a {@code Person}'s {@code Name} matches any of the keywords given.
+ */
+public class RoomIsOccupiedPredicate implements Predicate<Room> {
+
+    @Override
+    public boolean test(Room room) {
+        return room.getVacancy().equals(Vacancy.OCCUPIED);
```

Comment:
Hi Pei Lin, the PR we just pushed means that you can just use !room.isVacant() !